### ISSUE TITLE
Request: Sticky offset

### ISSUE THREAD

**Issue by t-l13** [nebularg/Parrot2#68]
Requesting the ability to reposition where the Sticky text for effects like Pow etc. are relative to the center of the scroll area, instead of only smack dab in the center.

The current implementation mixes poorly with the parabola scrolling of normal text. The parabola cannot be positioned in peripheral view of the center of the screen because Sticky/Pow criticals stick too close to its center. Repositioning the scroll area to accomodate the Sticky positioning pushes the parabola text too far to the edge of the screen to be easily visible. 

Simply put, the ability to change the position of Sticky text relative to the normal text is needed for the parabolic scrolling effect to serve its purpose: Keeping the center of the screen clear, while still being in your peripheral vision.
